### DISCUSSION TITLE
Option to enable Windows Spotlight

### DISCUSSION THREAD

**Discussion by nemchik** [LeDragoX/Win-Debloat-Tools#146] [Ideas]
I personally like having the Windows Spotlight set as my desktop background and lock screen image. It would be very nice to have a checkbox in the GUI to easily toggle this (since it is disabled by default in https://github.com/LeDragoX/Win-Debloat-Tools/blob/b7b72c77b40974344dc6e955d3b3d65f4384f325/src/scripts/Optimize-Privacy.ps1#L222-L228 )

It's easy enough to revert, but it would be appreciated if it could be an option in the GUI.

**Comment by LeDragoX** [comment]
Your idea seems to fit the Issues tab. I'll add this later.

**Comment by LeDragoX** [comment]
Moved to #156.
